twake/backend/core/src/Twake/Discussion/Services/MessageSystem.php

```diff
@@ -556,6 +560,39 @@ public function save($object, $options, $user = null, $application = null)
         return $array;
     }
 
+    public function sendToNode($message){
+        $messageArray = $message->getAsArray();
+
+        $channelDetails = $this->doctrine->getRepository("Twake\Core:CachedFromNode")->findOneBy(Array("company_id" => "unused", "type" => "channel", "key"=>$messageArray["channel_id"]));
+        if($channelDetails){
+
+            $md2text_options = Array("keep_mentions" => true);
+            $text_content = $this->mdToText($message->getContent(), $md2text_options);
+            preg_match_all("/@[^: ]+:([0-f-]{36})/m", $text_content, $users_output);
+            preg_match_all("/(^| )@(all|here|channel|everyone)[^a-z]/m", $text_content, $global_output);
+            $mentions = [
+                "users" => $users_output[1],
+                "special" => $global_output[1]
```

Comment:
`specials` please

Reply:
Oups, yes totally "specials" not "special"